gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -262,11 +264,21 @@ def main(args):  # noqa: D103
                 overwrite=args.overwrite,
             )
 
-        # TODO: Need to use annotate_subpop_meta before subpop assignment
         if args.assign_subpops:
-            raise NotImplementedError(
-                "Sub-population assignment is not currently implemented."
+            logger.info("Assigning subpops...")
+            pop_pca_scores_ht = ancestry_pca_scores(
```

Comment:
see comment in `assign_pops`. Alternatively, we can just change it so that no matter what, the HT is passed in to `assign_pops`? I just don't like different functionality for the global pops vs subpops